src/coreclr/scripts/coreclr_arguments.py

```diff
@@ -264,7 +264,7 @@ def check_and_return_default_product_location(product_location):
         self.verify(args,
                     "core_root",
                     check_and_return_default_core_root,
-                    "Error, incorrect core_root location.")
+                    "Error, Core_Root could not be determined, or points to a location that doesn't exist.")
```

Comment:
The original message confused me when I saw it, so I tried to write something better.